### ISSUE TITLE
'dummy' and 'stdout' schemes dont work

### ISSUE THREAD

**Issue by light** [getsentry/raven-ruby#897]
This configuration :
``` ruby
Raven.configure do |config|
  config.scheme = 'dummy'
end
```
results in :
```
ERROR -- sentry: ** [Raven] Unable to record event with remote Sentry server (NameError - uninitialized constant Raven::Transports::Dummy):
/usr/local/bundle/gems/sentry-raven-2.9.0/lib/raven/client.rb:62:in `transport'
/usr/local/bundle/gems/sentry-raven-2.9.0/lib/raven/client.rb:43:in `send_event'
/usr/local/bundle/gems/sentry-raven-2.9.0/lib/raven/instance.rb:81:in `send_event'
/usr/local/bundle/gems/sentry-raven-2.9.0/lib/raven/instance.rb:126:in `capture_type'
...
```

**Comment by geoffwa** [comment]
Also saw this - manually adding a

**Comment by st0012** [comment]
Thanks for reporting! I've fixed it in #986
